examples/hitl/rearrange_v2/app_state_feedback.py

```diff
@@ -0,0 +1,158 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Meta Platforms, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+from typing import Final, List, Optional
+
+from app_data import AppData
+from app_state_base import AppStateBase
+from app_states import (
+    create_app_state_cancel_session,
+    create_app_state_load_episode,
+)
+from session import Session
+from util import get_top_down_view
+
+from habitat_hitl.app_states.app_service import AppService
+from habitat_hitl.core.key_mapping import KeyCode
+from habitat_hitl.core.text_drawer import TextOnScreenAlignment
+from habitat_hitl.core.ui_elements import HorizontalAlignment
+from habitat_hitl.core.user_mask import Mask
+
+OK_BUTTON_ID = "feedback_ok"
+SCREEN_TIMEOUT = 30.0
+
+FONT_SIZE_LARGE: Final[int] = 32
+FONT_SIZE_SMALL: Final[int] = 24
+
+
+class AppStateFeedback(AppStateBase):
```

Comment:
@zephirefaith We can use this state in place of `app_state_skip_episode` in `phase3`.